medusa/providers/nzb/anizb.py

```diff
@@ -118,5 +121,32 @@ def _get_size(self, item):
         """Override the default _get_size to prevent it from extracting using the default tags."""
         return try_int(item.get('size'))
 
+    def _get_episode_search_strings(self, episode, add_string=''):
+        """Get episode search strings."""
+        _ = add_string
```

Comment:
It's overriding method. But where not using add_string. If you leave out the _ = add_string, you get unused variable.